### DISCUSSION TITLE
Can Thumb cluster use stabilizers

### DISCUSSION THREAD

**Discussion by Looooopy** [joshreve/dactyl-keyboard#59] [Q&A] [answered]
Is it possible to generate Thumb cluster with keys that use stabilizers like he is using in this article?
https://kbd.news/Dactyl-Stabilized-831.html

Or is it overkill to have stabilizers for the large keys?

**Comment by joshreve** [comment, marked as answer]
I think it is overkill for 1.5U.  If someone creates a cluster using 2U, it can certainly be done.  It really wouldn't be too bad to do since some clusters already have custom shapes for the 1.5U keys.  This would be an additional step for whoever customizes the thumb cluster.  I don't have a plan to create a cluster with a 2U at this time.  Adding the stabilizers is much less work than creating a thumb cluster with webs and walls, especially if you know where to inject the key plate.

If someone worked to get a cluster with a 2U laid out and needed some help, I'd be happy to provide support getting a specific key plate with stabilizers in place.
